### ISSUE TITLE
`extends` should not enforce a type Symbol

### ISSUE THREAD

**Issue by vfonic** [Shopify/theme-check#553]
I'm trying to add "global" `.theme-check.yml` for all my Shopify themes/projects.
I'd like to require that one `.theme-check.yml` in each project's root `.theme-check.yml`.
Does this functionality exist?

Is this what I'm looking for:
https://github.com/Shopify/theme-check/blob/main/config/default.yml#L3

How can I use this?

I tried this:

```yml
root: .

extends: "/Users/viktor/.theme-check.yml"
```

But then I get this error:
> bad configuration type for extends: expected a Symbol, got "/Users/viktor/.theme-check.yml"

I think this comment here might be related:
https://github.com/Shopify/theme-check/blob/main/lib/theme_check/config.rb#L70-L73

So to sum it up, this doesn't seem to be currently possible, correct?

**Comment by charlespwd** [comment]
Oh that's a bug. It's supposed to work :upside_down_smile:
